src/components/ControlledInputField/ControlledInputField.js

```diff
@@ -0,0 +1,60 @@
+import { makeStyles, TextField } from "@material-ui/core"
+import { Controller } from "react-hook-form"
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    display: "flex",
+    flexDirection: "column",
+    fontSize: theme.typography.fontSize,
+    fontFamily: theme.typography.fontFamily,
+
+    "& .MuiFormHelperText-contained": {
+      margin: "0",
+    },
+
+    "& .MuiOutlinedInput-input": {
+      padding: "6px 10px 6px 10px",
+      fontSize: theme.typography.fontSize,
+    },
+  },
+  label: {
+    fontWeight: theme.typography.fontWeightBold,
+    paddingBottom: "4px",
+  },
+}))
+
+const ControlledInputField = ({ control, name, label, placeholder, type }) => {
```

Comment:
I see you are using the same component for both text and numbers, I would actually advise against this, can you make a separate number input component